.github/workflows/integration.yml

```diff
@@ -218,3 +218,11 @@ jobs:
           RUNTIME_TYPE: ${{ matrix.run.runtimeType }}
           TEST_USERNS: ${{ matrix.run.userns }}
           CONTAINER_DEFAULT_RUNTIME: ${{ matrix.run.defaultRuntime }}
+      # Codecov
+      - run: sudo go tool covdata textfmt -i=build/coverage/bats -o build/coverage/coverprofile
```

Comment:
very small nit, definitely ignorable: I don't love that we're hardcoding build/coverage/bats in two places. it's not a huge deal, but we could define an env variable and pass it in here and in the `make localintegration` call, right? then not need to hardcode in test_runner.sh?